linux/lvm.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/anuvu/disko"
 )
 
+const pvMetaDataSize = 128 * disko.Mebibyte
+const thinMetaLVSize = 1024 * disko.Mebibyte
```

Comment:
is this the Pool size mentioned in the commit message?

Reply:
Yes?

> The MetaData pool size (--poolmetadatasize=) is hard coded to
> 1GiB and a spare pool is created (--poolmetadataspare=y).